index.html

```diff
@@ -227,5 +235,29 @@ <h1>Skills</h1>
             </div>
         </div>
     </section>
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form action="https://formspree.io/f/xrgvqloo" method="POST">
+                <input type="text" name="tect" maxlength="30" placeholder="Full name" required>
+                <input type="email" name="email" placeholder="Email adress" required>
+                <textarea maxlength="500" name="message" placeholder="Enter text here" required></textarea>
+                <button type="submit" name="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] Kindly increase the padding for the inputs and the `Get in touch` button to match the Figma design.